Werewolf for Telegram/Database/AchievementsReworked.cs

```diff
@@ -200,6 +200,8 @@ public enum AchievementsReworked
         GoodChoiceForYou = 92,
         [Display(Name = "Increase the Pack!"), Description("After the wolf cub died, infect 2 players as the alpha wolf!")]
         IncreaseThePack = 93,
+        [Display(Name = "The Firefighter!"), Description("As the guardian angel clean At least two homes from kerosene of The arsonist.")]
```

Comment:
The casing here is kind of weird. 
```suggestion
        [Display(Name = "The Firefighter!"), Description("As the Guardian Angel, clean at least two homes from the Arsonist's kerosene.")]
```